Casks/j/jetdrive-toolbox.rb

```diff
@@ -0,0 +1,47 @@
+cask "jetdrive-toolbox" do
+  version "4.5"
+  sha256 "2a52a2d24d22a120c737e5d9a13eceadd7be59751185a0821ef84fdf6f5605f8"
+
+  url "https://cdn.transcend-info.com/files/special/JetDriveToolbox_v#{version}.zip",
+      user_agent: :fake
+  name "JetDrive Toolbox"
+  desc "Helper for Transcend SSDs and expansion cards"
+  homepage "https://www.transcend-info.com/Support/Software-181/"
+
+  livecheck do
+    url :homepage
+    regex(%r{<p>MAC</p>\s*<p>Version\s*(\d+(?:\.\d+)*)</p>}i)
+    strategy :page_match do |page, regex|
+      page.scan(regex).map(&:first)
+    end
```

Comment:
```suggestion
    url "https://www.transcend-info.com/Software/1151/"
    strategy :header_match
```

It seems that we can avoid using a `regex` by using the download URL from the homepage.

Reply:
Thanks, I had no idea that such option even exists 💪